nerdlets/cloud-optimize-nerdlet/cloud-optimize.js

```diff
@@ -130,18 +155,33 @@ export default class CloudOptimize extends React.Component {
         })
     }
 
-    fetchAwsPricing(region){
+    fetchCloudPricing(){
+        let { config, cloudData, cloudRegions } = this.state
+        
         return new Promise((resolve)=>{
-            console.log(`fetching aws ec2 pricing: ${region}`)
-            // https://cors.io/?https://a0.p.awsstatic.com/pricing/1.0/ec2/region/${project.awsRegion}/ondemand/linux/index.json
-            // cors hack
-            fetch(`https://yzl85kz129.execute-api.us-east-1.amazonaws.com/dev?url=https://a0.p.awsstatic.com/pricing/1.0/ec2/region/${region}/ondemand/linux/index.json`).then((response)=> {
-                return response.json()
-            }).then((myJson)=>{
-                this.setState({awsPricing: myJson})
+            let cloudPromises = Object.keys(config.cloudData).map((cloud)=>{
+                return new Promise((resolve)=>{
+                    fetch(`https://nr1-cloud-optimize.s3-ap-southeast-2.amazonaws.com/${cloud}/regions.json`).then((response) => {
```

Comment:
Should have a config var for the s3 bucket, so folks can store it themselves or point at their own instance of it when deploying to a new NR account. Is this small enough to store in NerdStore instead?

Reply:
Moved under #8 as feature enhancement